### ISSUE TITLE
Remove Moment as an example from this readme

### ISSUE THREAD

**Issue by TheLarkInn** [asfktz/autodll-webpack-plugin#85]
When you DLL you cannot tree-shake code from the DLL bundle. Since on average only 20% if momentjs is used when imported, its best to remove it from the README. Otherwise, keep up the great work on this!

**Comment by asfktz** [comment]
Hi, @TheLarkInn!
Thank you so much, I appreciate it (:

> Since on average only 20% if momentjs is used when imported, its best to remove it from the README.

I see.
Originally, I thought of it as a plugin to speed up development, so at the time it made sense to add momentjs to the DLL. over time I realized that people use it for production too, so yeah, I agree.
I'll remove it.

Btw, I wanted to ask you,
a few people asked us to support code splitting and the ProvidePlugin,
I saw that it's currently not supported by the DllPlugin and I wonder if there is some technical limitation to implement this or that it is something that can be solved with a PR.
If so, maybe we can help?
